torch/export/exported_program.py

```diff
@@ -489,31 +489,8 @@ def update_arg(old_arg, new_ph):
     # propagate names to higher order op subgraphs
     _name_hoo_subgraph_placeholders(gm)
 
-    # Run this pass before creating input/output specs, since size-related CSE/DCE might affect output signature.
-    # Overwrite output specs afterwards.
-    from torch._export.passes._node_metadata_hook import (
-        _node_metadata_hook,
-        _set_node_metadata_hook,
-    )
     from torch._functorch._aot_autograd.input_output_analysis import _graph_output_names
 
-    if not torch._dynamo.config.do_not_emit_runtime_asserts:
```

Comment:
@angelayi Do you know what exactly this here is needed for? I thought the runtime asserts emitted would just be the same as in the initial export call, but maybe this serves some specific purpose?